interfaces/builtin/fwupd.go

```diff
@@ -71,18 +84,49 @@ const fwupdPermanentSlotAppArmor = `
   /sys/firmware/efi/efivars/ r,
   /sys/firmware/efi/efivars/** rw,
 
+  /proc/modules r,
+  /run/udev/data/* r,
+  /sys/kernel/security/lockdown r,
+  owner @{PROC}/@{pid}/mountinfo r,
+  /dev/cpu/*/msr rw,
+
+  owner @{PROC}/@{pid}/mounts r,
+  /run/mount/utab r,
+  # Realtek MST plugin
+  /dev/i2c-[0-9]* rw,
```

Comment:
nitpick, shouldn't this go below with the rest of /dev accesses ?

Reply:
I have moved things around. Also added test for StaticInfo.